create/create.js

```diff
@@ -10,27 +10,38 @@ const logoutButton = document.getElementById('logout');
 
 form.addEventListener('submit', async e => {
     // prevent default
-
+    e.preventDefault();
     // get the name and family id from the form
+    const formData = new FormData(form);
 
+    const family = formData.get('family-id');
+    const bunName = formData.get ('bunny-name');
     // use createBunny to create a bunny with this name and family id
-    
+    await createBunny({
+        name: bunName,
+        family_id: family
+    });
+    console.log(bunName);
     form.reset();
```

Comment:
We will also want to redirect to the families page after we create the bunny and reset the form.